gnd/src/main/java/com/google/android/gnd/ui/map/gms/GoogleMapsMapAdapter.java

```diff
@@ -86,6 +100,32 @@ public GoogleMapsMapAdapter(GoogleMap map, Context context) {
     onCameraMove();
   }
 
+  public void renderJsonLayer() {
+    File file = new File(context.getFilesDir(), GEO_JSON_FILE);
+    if (file.exists()) {
```

Comment:
Rather than nesting the whole conditional block, fail fast if file doesn't exist (Log.e please)